docs/working-model/organizational-structure/catena-x/catena-x-office.md

```diff
@@ -1,16 +1,15 @@
 ---
 sidebar_position: 5
+title: Catena-X office
```

Comment:
I think we should keep the h1 heading - as this is used by all the other documents as well - at leas I don't see a benefit in using title?

Reply:
we are usually using title instead of h1

Reply:
Is there a benefit? otherwise i think we should keep it consistent across all documents?